docs/develop/sdk/overview.mdx

```diff
@@ -53,48 +52,10 @@ Depending on the deployment target, add one of the following dependencies:
 
 To use Jackson for serialization, add the following dependency: `dev.restate:sdk-serde-jackson:VAR::JAVA_SDK_VERSION`
 
-Now you need to configure the protobuf plugin to build your Protobuf contracts. For example, with Gradle (Kotlin script):
+Now you need to configure the protobuf plugin to build your Protobuf contracts.
+Have a look at the [`build.gradle.kts` file of the Java Gradle template](https://github.com/restatedev/examples/blob/main/templates/java-gradle/build.gradle.kts) for an example.
```

Comment:
I really like the new 1 sentence per line style :-) This makes adding comments a lot easier.